src/PortingAssistantExtensionClientShared/PortingAssistantLanguageClient.cs

```diff
@@ -52,8 +52,23 @@ class VisualBasicContentDefinition
         internal static FileExtensionToContentTypeDefinition VbFileExtensionDefinition;
     }
 
-    [ContentType("CSharpFileType")]
-    [ContentType("VisualBasicFileType")]
+    class PAcContentDefinition
```

Comment:
Not sure if it matters but there's an extra c here

Reply:
thx, will fix typo